src/components/TextField/__snapshots__/TextField.test.tsx.snap

```diff
@@ -13,7 +13,7 @@ exports[`<TextField /> with all options matches its snapshot 1`] = `
   maxLength={100}
   multiline={false}
   onBlur={[MockFunction onBlur]}
-  onChange={[Function]}
```

Comment:
Hm... isn't `onChanged` the one that's deprecated in Fabric's `TextField`?

https://github.com/OfficeDev/office-ui-fabric-react/pull/5764

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts#L167

Reply:
NEVERMIND... this diff is a revert of @swese44 's attempt to move us over to `onChange`.